src/app/file-browser/components/edit-tags/edit-tags.component.ts

```diff
@@ -33,11 +53,15 @@ export class EditTagsComponent implements OnInit, DoCheck, OnDestroy, HasSubscri
 
   public isEditing = false;
 
+  public newTagInputError = false;
```

Comment:
Can we reset this value when you open/close the tag editor? I feel like once the error appears, I should be able to make it go away by just closing the editor, but when I re-open it, the error is still there and it feels like something's wrong.

On a related note, when this error is being displayed, you can switch to another file in your archive, open the tag editor on it, and the error will still be present which I think definitely shouldn't be the case.